rust/main/chains/hyperlane-ton/src/.gitignore

```diff
@@ -0,0 +1,5 @@
+runtime_test.rs
+main.rs
+mailbox_test.rs
+ism_test.rs
+announce_test.rs
```

Comment:
I wonder how these files are generated and why we would like to avoid committing them into source control?

Reply:
Removed